Parsers/ASimAuthentication/Parsers/ASimAuthenticationOktaSSO.yaml

```diff
@@ -1,19 +1,19 @@
 Parser:
-  Title: ASIM Okta identity management authentication parser
+  Title: Authentication ASIM parser for Okta identity management
```

Comment:
for Okta identity management -> for Okta

Reply:
should we change the file name and function name? How do we call the built in?